packages/api/test/helper/constants.ts

```diff
@@ -19,6 +19,17 @@ const activityLogs = [
     },
 ];
 
+const queryTemplates = [
+    {
+        name: "abc",
+        description: "lorem ipsum dolor amet",
+        resource: "61c9483bcd1cf99c3a211977",
+        app: "61c93a931da4a79d3a109947",
+        content: "abcd",
```

Comment:
And the content could be `SELECT * FROM users as u LEFT JOIN orders as o ON u.id=o.userId`